Readme.md

```diff
@@ -14,49 +14,38 @@ interactions without compromising either performance or ease of use.
 - [API Reference &rarr;](https://deno.land/x/capi/mod.ts)<br />A generated API
   reference, based on type signatures and TSDocs.
 
-## At a Glance
+## Installation
 
-Run the Capi development server.
+We can substitute the URL's target (in this case the WebSocket URL of a proxy).
+
+### [Node.js](https://nodejs.org/)
 
 ```sh
-deno run -A https://deno.land/x/capi/main.ts
+npm i https://capi.dev/frame/wss/rpc.polkadot.io/@latest/pkg.tar
 ```
 
-Create an import map with the specifier corresponding to your target.
-
-`import_map.json`
+### [Deno](https://deno.land/)
 
-```json
-{
-  "imports": {
-    "@capi/polkadot/": "http://localhost:4646/frame/wss/rpc.polkadot.io/@latest/"
-  }
-}
+```ts
+import {} from "https://capi.dev/frame/wss/rpc.polkadot.io/@latest/mod.js"
```

Comment:
This should use the import map IMO

Reply:
Good point

Reply:
Actually, that isn't the case (bc the extension).

Reply:
True. Still, I think it's cleaner as an import map – we want our readme to show the expected use.

Reply:
Sounds good. In that case, is it worth import-mapping directly to the `mod.js`? That way we can keep it extension-free after all.

Reply:
Or I suppose this isn't the intended use, as we want people to be able to easily access files within the codegened dir (such as guards, factories, misc.)